src/OnboardingSPA/components/ProgressBarSiteGen/index.js

```diff
@@ -0,0 +1,15 @@
+const ProgressBarSiteGen = ( { current, total } ) => {
+	const percentage = ( current / total ) * 100;
+	return (
+		<div className="nfd-progress-bar-site-gen">
```

Comment:
I see a lot of discrepancy in class names? `nfd-*-site-gen`, `nfd-sitegen-*`? 
Let's decide on one prefix?